bindings/C/c/adios2_c_variable.cpp

```diff
@@ -200,7 +200,10 @@ adios2_error adios2_variable_name(char *name, size_t *size,
         const adios2::core::VariableBase *variableBase =
             reinterpret_cast<const adios2::core::VariableBase *>(variable);
         *size = variableBase->m_Name.size();
-        variableBase->m_Name.copy(name, *size);
+        if (name)
```

Comment:
Let's use `if(name != nullptr)` to remove ambiguity

Reply:
I can do that if that's coding style, but there's no ambiguity in this statement.

Reply:
is `name` a nullptr or a value other than 0?